src/lib/components/Timeline/TimelineTitle.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC, PropsWithChildren, ComponentProps } from 'react';
+
+export type TimelineTitleProps = PropsWithChildren<ComponentProps<'h3'>>;
+
+export const TimelineTitle: FC<TimelineTitleProps> = ({ children, ...props }) => {
+  return (
+    <h3 className="text-lg font-semibold text-gray-900 dark:text-white" {...props}>
```

Comment:
This needs to be morphable into any heading level

Reply:
Should I make a prop called `tag` that is passed to the heading level. The user can then only pass in a range from 1-6 to the `tag` prop?

Reply:
Its a pretty common pattern to provide an as prop, like:

as="h5"